### PR TITLE
Feature/remove console logs story

### LINKED ISSUE #1847: Remove all console.logs() from stories and demos
In components like `inputfile` and many more console.log() are being used. Due the fact that not everybody has his console open (or even knows that a console exists), all logs should be printed onto the page.

### PR DESCRIPTION
**Pull request by 01231**
Fixes #1847

# Done is when (DoD):
- [x] Code selfreview has been performed.
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Modifications follow our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: src/components/10-atoms/checkbox/react/story.jsx [hunk 1]
```diff
@@ -33,18 +34,25 @@ storiesOf('Examples/Checkbox/React', module)
...
-        disabled={disabled}
-        variant={variant}
-        required={required}
-        styled={styled}
-        onChange={e =>
-          console.log('checkbox', name, ' changed to: ', e.target.checked)
-        }
-        error={errortext ? 'Please accept our terms and conditions.' : ''}
-      />,
+      <>
```
Comment (raphaellueckl) [inline src/components/10-atoms/checkbox/react/story.jsx:37]: ?

Comment (01231) [reply src/components/10-atoms/checkbox/react/story.jsx:37]: https://medium.com/@rossbulat/the-empty-tag-is-shorthand-for-react-fragment-6129e70e418a

File: src/components/30-organisms/footer/react/story.jsx [hunk 2]
```diff
@@ -6,7 +6,6 @@ import Changelog from '../CHANGELOG.md';
 import AXAFooterReact from './Footer';
 
 const setNewlyClickedLink = link => {
-  console.log('clickediclick');
```
Comment (danyball) [inline src/components/30-organisms/footer/react/story.jsx]: :D

**Review by danyball — COMMENTED** [review]
Will check usability if feature deploy is done. 

Do we have no console logs at the story-files or the js demos?

**Comment by 01231** [comment]
@danyball yes we have no console.log, because we can not expect our users to have the console open

**Comment by 01231** [comment]
icon test is failing :(

**Comment by danyball** [comment]
> @danyball yes we have no console.log, because we can not expect our users to have the console open

No, i mean, did you search for logs globally?

**Comment by 01231** [comment]
> > @danyball yes we have no console.log, because we can not expect our users to have the console open
> 
> No, i mean, did you search for logs globally?

Yes I did. Globally in all stories and demos. No they're all gone as expected.

**Review by raphaellueckl — APPROVED** [review]
naise

**Review by danyball — APPROVED** [review]

**Pull request merged**